x-pack/metricbeat/module/azure/data.go

```diff
@@ -84,7 +84,7 @@ func EventsMapping(metrics []Metric, metricset string, report mb.ReporterV2) err
 				event, metricList = createEvent(timestamp, defaultMetric, groupTimeValues)
 			}
 			if metricset == nativeMetricset {
-				event.ModuleFields.Put("metrics", metricList)
+				event.ModuleFields.Put(fmt.Sprintf("%s.metrics", nativeMetricset), metricList)
```

Comment:
Would this work the same?
```suggestion
				event.MetricSetFields.Put("metrics", metricList)
```

Reply:
reverted change